README.md

```diff
@@ -1,6 +1,8 @@
 # IFC for Revit (revit-ifc)
 
-This is the .NET code used by Revit 2019 and Revit LT 2019 to support IFC. The open source version can override the version that comes standard with shipped Revit. This contains the source code for Link IFC, IFC export and the IFC export UI.
+This is the .NET code used by Revit 2019, Revit 2020, Revit LT 2019 and Revit LT 2020 to support IFC.
+The open source version can override the version that comes standard with shipped Revit.
+This contains the source code for Link IFC, IFC export and the IFC export UI.
```

Comment:
Actually, and Revit 2021.  @WawanSolihin , can you make a JIRA item to update this?